bindings/test/unit/test_server.py

```diff
@@ -210,9 +210,6 @@ def test___init___defaults(self):
         self.assertTrue(isinstance(connection.server_wrapper, server.HTTPSServerWrapper))
         self.assertEqual(connection.server_wrapper.pulp_connection, connection)
         self.assertEqual(connection.verify_ssl, True)
-        self.assertEqual(connection.ca_path, server.DEFAULT_CA_PATH)
-        # 1142376 - verify default path points to a known valid file
-        self.assertEqual(server.DEFAULT_CA_PATH, '/etc/pki/tls/certs/ca-bundle.crt')
```

Comment:
These assertions could probably be adapted rather than removed.

For line #213, it can maybe be changed to `self.assertTrue(connection.ca_path in server.DEFAULT_CA_PATH_LIST)`

For line #215, it seems important to have something verifying that the default CA path is set to a file that actually exists on a given platform. A new unit test covering the behavior of `get_from_default_paths` is necessary to ensure that that function does not return `None`, so that the tests appropriately fail if/when some distribution doesn't place the CA bundle in a location known to pulp.